src/status_im/chat/handlers/commands.cljs

```diff
@@ -43,8 +43,10 @@
                 callback #(let [result (get-in % [:result :returned])
                                 result' (if (:markup result)
                                          (update result :markup cu/generate-hiccup)
-                                         result)] 
-                            (dispatch [:set-in [:message-data data-type message-id] result'])
+                                         result)]
+                            ;; don't fill message data with nil results
```

Comment:
Your PR comment is great! It contains much more info than here. 
Maybe add the important details from the PR message? When debugging we certainly won't dig PR history while comments will be easily available.